lte/gateway/c/oai/tasks/nas5g/src/M5GDLNASTransport.cpp

```diff
@@ -143,6 +143,13 @@ int DLNASTransportMsg::EncodeDLNASTransportMsg(
     return encoded_result;
   else
     encoded += encoded_result;
+  if ((encoded_result =
+           dl_nas_transport->pdu_session_identity.EncodePDUSessionIdentityMsg(
+               &dl_nas_transport->pdu_session_identity, PDU_SESSION_IDENTITY,
+               buffer + encoded, len - encoded)) < 0)
+    return encoded_result;
+  else
+    encoded += encoded_result;
```

Comment:
Add braces around single statement conditional blocks

```suggestion
  if ((encoded_result =
           dl_nas_transport->pdu_session_identity.EncodePDUSessionIdentityMsg(
               &dl_nas_transport->pdu_session_identity, PDU_SESSION_IDENTITY,
               buffer + encoded, len - encoded)) < 0) {
    return encoded_result;
  } else {
    encoded += encoded_result;
    }
```

Reply:
Added braces for this file, we will fix for remaing all files in separate PR